Completed miscellaneous functions.

MagicalDrizzle opened the pull request:
Just completed a bunch of things that I saw missing in there.
Also changed some of the wording and description.

Sparen commented:
Please specify what changes you made other than capitalizing X and Y.

In addition, you copied `RaiseError` twice - I believe the latter is supposed to be `assert`?

MagicalDrizzle commented:
I completed the documentation for the CreateShot functions - a bunch of them is missing. Also changed the wording on DeleteShotAll and DeleteShotInCircle to match yours.
Added pi() in Math functions.
Added StartShotScript() in Shot functions.
And yes, it's supposed to be `assert`. I forgot to change it.

drakeirving commented:
These diffs ended up a bit strange because you split it into delete/upload for some reason but it seems to be fine?
